mllib/src/test/scala/org/apache/spark/ml/recommendation/ALSSuite.scala

```diff
@@ -701,7 +701,7 @@ class ALSSuite extends MLTest with DefaultReadWriteTest with Logging {
         }.getMessage.contains(msg))
         assert(intercept[StreamingQueryException] {
           testTransformer[A](dataFrame, model, "prediction") { _ => }
-        }.getMessage.contains(msg))
+        }.getCause.getCause.getMessage.contains(msg))
```

Comment:
Could you use your new helper method here? Calling `getCause` like that feels a bit weird.